python/lsst/ts/observing/utilities/auxtel/latiss/getters.py

```diff
@@ -0,0 +1,74 @@
+import time
+import asyncio
+import lsst.daf.persistence as dafPersist
+import logging
+from lsst.rapid.analysis import BestEffortIsr
+
+STD_TIMEOUT = 10  # seconds
+
+logging.basicConfig()
```

Comment:
If you are in creating the `ImageGetter` class, we can probably remove this log handling here.